src/components/books/form.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-vars */
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../redux/books/books';
+
+function Form() {
+  const dispach = useDispatch();
+  const addHandler = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: uuidv4(),
+      bookTitle: e.target.title.value,
+      bookCategory: e.target.category.value,
+    };
+    console.log('ID');
+    console.log(newBook.id);
```

Comment:
**Optional Changes**
Your `form` is nicely done. Bravo! However;
- [ ] It's good practice to allows delete `console.log` codes before pushing into `git/version` control. Let as remove these. Many devs have different opinions, but at least most of them agree that you shouldn't push to version control.